README.md

```diff
@@ -159,7 +164,9 @@ The resulting image will be found in `yocto/build/tmp/deploy/image/<machine>`, h
 
 ### Flashing an Image <a name="flash"></a>
 
-The internal storage of the products Charge Control C (Tarragon) and M (EVAcharge SE) is divided into several partitions and uses RAUC to handle updates. If you are interested in adding RAUC to your own image you can take a look at RAUC's documentation https://rauc.readthedocs.io/en/latest/integration.html#yocto .To flash an image it is important to identify the currently not in use partition because this will be our target partition. The following tables describe the partitioning for each Charge Control M and Charge Control C:
+The internal storage of the products Charge Control C (Tarragon) and M (EVAcharge SE) is divided into several partitions and uses RAUC to handle updates.
```

Comment:
I think this whole paragraph can be also extended to include info about Charge SOM.

Reply:
I slightly changed the paragraph